criu/seize.c

```diff
@@ -645,7 +645,7 @@ static int collect_threads(struct pstree_item *item)
 	/* The number of threads can't be less than already frozen */
 	item->threads = xrealloc(item->threads, nr_threads * sizeof(struct pid));
 	if (item->threads == NULL)
-		return -1;
```

Comment:
we need to restore a previous value of item->threads

Reply:
Sorry, I do not understand what you mean? Should I just drop this?

Reply:
if item->thread was not null, we will have a memory leak. Should we do something like this:
tmp = xrealloc(....)
if (tmp == NULL)
  return -1;
item->thread = tmp;

Reply:
fixed